beacon-chain/db/kv/state.go

```diff
@@ -303,6 +303,30 @@ func (s *Store) saveStatesEfficientInternal(ctx context.Context, tx *bolt.Tx, bl
 			if err := valIdxBkt.Put(rt[:], validatorKeys[i]); err != nil {
 				return err
 			}
+		case *ethpb.BeaconStateCapella:
+			var pbState *ethpb.BeaconStateCapella
+			var err error
+			pbState, err = statenative.ProtobufBeaconStateCapella(rawType)
```

Comment:
Other states have `if features.Get().EnableNativeState` statements here, but I skipped it for Capella.